reporters/ClickhouseReporter.js

```diff
@@ -100,8 +100,12 @@ const TABLE_DEFINITIONS = [
  */
 function santizeCallArgs(args) {
     // in some cases call args have been stringified, so unwrap that first.
-    const argsArray = typeof args === 'string' ? JSON.parse(args) : (args || []);
-    return argsArray.map((/** @type {string} */ s) => s.replace(/'/g, ''));
+    try {
```

Comment:
ditto . When this can happen?

Reply:
I removed this, as I couldn't repro the issue that triggered the error.